packages/suite/src/actions/wallet/__fixtures__/coinjoinClientActions.ts

```diff
@@ -17,11 +18,12 @@ export const onCoinjoinRoundChanged = [
         },
         params: {
             phase: 0,
-            inputs: [{ accountKey: 'b' }, { accountKey: 'c' }],
+            inputs: [{ accountKey: 'a' }, { accountKey: 'c' }],
```

Comment:
why this change? this is a valid test which checks that `COINJOIN.SESSION_ROUND_CHANGED` was not called

Reply:
fixed